### PR TITLE
Create numbers.txt

### PR DESCRIPTION
**Pull request by ghost**
wooo wooo

### CODE REVIEW

File: numbers.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+one
+two
+seventeen
```
Comment (ttd2089) [inline numbers.txt]: I think this should be "three"

Comment (ghost) [reply numbers.txt]: good call, I'll fix that up

Comment (ghost) [reply numbers.txt]: fixed

**Comment by ttd2089** [comment]
this junk

File: b.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+bbbbb
```
Comment (ttd2089) [inline b.txt:1]: more junk

Comment (ttd2089) [reply b.txt:1]: reply

**Comment by ttd2089** [comment]
anther here

File: d.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+ddddd
```
Comment (ttd2089) [inline d.txt:1]: adhada

**Pull request closed without merging**